### PR TITLE
Developers SDK Page

### PR DESCRIPTION
**Pull request by aaronmgdr**
### Description

creates `/developers` page 
Screen Shot 2020-02-14 at 4 18 09 PM
Screen Shot 2020-02-14 at 4 18 18 PM
Screen Shot 2020-02-14 at 4 18 27 PM

### Tested
Screen Shot 2020-02-14 at 4 17 59 PM

### Other changes

changes menu to be `About - Join - Validators - Developers - Community `

changes /build to /validators

### Related issues

- resolves https://github.com/celo-org/celo-labs/issues/249

### CODE REVIEW

**Comment by tjlahey** [comment]
Lookin' real good. I'm trying to catch anything but looking real good thus far.

File: packages/web/src/dev/DevelopersPage.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import Features from 'src/dev/Features'
+import FullStack from 'src/dev/FullStack'
+import Sandbox from 'src/dev/Sandbox'
+import { H2, H3 } from 'src/fonts/Fonts'
+import OpenGraph from 'src/header/OpenGraph'
+import { I18nProps, NameSpaces, withNamespaces } from 'src/i18n'
+import ConnectionFooter from 'src/shared/ConnectionFooter'
+import menuItems from 'src/shared/menu-items'
+import { standardStyles } from 'src/styles'
+const previewImage = require('src/dev/dev-og.png')
```
Comment (jmrossy) [inline packages/web/src/dev/DevelopersPage.tsx]: Why require here? Also, spacing

File: packages/web/src/dev/Engage.tsx [hunk 2]
```diff
@@ -141,8 +43,8 @@ interface ContentProps {
 const Content = React.memo(function _Content(props: ContentProps) {
   return (
     <View style={styles.paragraphArea}>
-      <H3 style={standardStyles.elementalMarginBottom}>{props.noun}</H3>
-      <H4 style={standardStyles.elementalMarginBottom}>{props.verb}</H4>
+      {props.noun && <H3 style={standardStyles.elementalMarginBottom}>{props.noun}</H3>}
```
Comment (jmrossy) [inline packages/web/src/dev/Engage.tsx]: These prop name seem odd to me. How about `h3Text` and `h4Text`

File: packages/web/src/dev/Engage.tsx [hunk 3]
```diff
@@ -206,6 +108,136 @@ const Aside = React.memo(function _Aside({
   )
 })
 
+export function Contribute() {
+  const { t } = useTranslation(NameSpaces.dev)
+  return (
+    <GridRow
+      allStyle={[standardStyles.elementalMarginBottom, styles.alignOut]}
```
Comment (jmrossy) [inline packages/web/src/dev/Engage.tsx]: It's probably a negligible improvement but in the app we avoid style arrays where we can b.c. they create new instances each render, breaking react's diff checking and forcing a rerender.
Instead we would destructure `elementalMarginBottom` in `styles.alignOut` like
```
...standardStyles.elementalMarginBottom,
other styles
```

File: packages/web/src/dev/FullStack.tsx [hunk 4]
```diff
@@ -221,16 +222,22 @@ class FullStack extends React.PureComponent<I18nProps & ScreenProps, State> {
               <Li style={textStyles.invert}>{t('proof.onChain')}</Li>
             </StackSection>
             <View
+              // @ts-ignore
```
Comment (jmrossy) [inline packages/web/src/dev/FullStack.tsx]: why dis?

File: packages/web/src/dev/Sandbox.tsx [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+              </View>
+            </View>
+          </View>
+        </Cell>
+      </GridRow>
+    </View>
+  )
+}
+
+async function trackSandBox() {
```
Comment (jmrossy) [inline packages/web/src/dev/Sandbox.tsx]: nit: `trackSandBoxClick`

**Review by jmrossy — APPROVED** [review]
Overall looks good! Just some minor optional comments

File: packages/web/src/dev/FullStack.tsx (hunk 4 above)
Comment (aaronmgdr) [reply packages/web/src/dev/FullStack.tsx]: was an issue with using transition as a property , fixed it up now

File: packages/web/src/dev/DevelopersPage.tsx (hunk 1 above)
Comment (aaronmgdr) [reply packages/web/src/dev/DevelopersPage.tsx]: at one point i found that import and images don't always work out and it got me stuck in this habit  even if its not true anymore

File: packages/web/src/dev/Engage.tsx (hunk 3 above)
Comment (aaronmgdr) [reply packages/web/src/dev/Engage.tsx]: oh interesting that makes sense

Comment (aaronmgdr) [reply packages/web/src/dev/Engage.tsx]: when i did this 
```
const styles = Stylesheet.create({
nameOfStyle: {
...standardStyles.centered,
width: 100
}}
```
`style={styles.nameOfStyle}`

the centered styles were not applied. this happened every time i spread a style

**Pull request merged**